app/assets/stylesheets/schedules.css

```diff
@@ -0,0 +1,11 @@
+/* app/assets/stylesheets/schedules.css 
+This contains the css required for all the schedules views*/
+.container {
```

Comment:
Calling a class container may cause conflicts with the bootstrap container class.

Reply:
Good point. I have removed this